storage/replica.go

```diff
@@ -902,8 +901,15 @@ func (r *Replica) proposeRaftCommand(ctx context.Context, ba roachpb.BatchReques
 		OriginReplica: *replica,
 		Cmd:           ba,
 	}
-	cmdID := ba.GetOrCreateCmdID(r.store.Clock().PhysicalNow())
-	idKey := makeCmdIDKey(cmdID)
+
+	makeCmdIDKey := func() cmdIDKey {
+		buf := make([]byte, 0, 16)
+		buf = encoding.EncodeUint64(buf, uint64(r.store.Clock().PhysicalNow()))
```

Comment:
Now that `cmdIDKeys` are not used as rocksdb keys, I don't think we need to have a time prefix. They can just be random.